scripts/terraform/dmitri_co.tf

```diff
@@ -8,19 +8,54 @@ provider "aws" {
   region = "us-east-1"
 }
 
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
```

Comment:
Let's figure out the right values for this and for https://github.com/dmitrinesterenko/sample_phoenix_elasticbeanstalk/pull/3/files#diff-730343b9bce1f8c1bbd67cb50606b5c7R20